client/src/Questions/QuestionsList.js

```diff
@@ -1,17 +1,20 @@
-import { Link } from 'react-router-dom';
+import { Link, useParams } from 'react-router-dom';
 
-// import './ResourcesList.scss';
 import { useAuthContext } from '../AuthContext';
 import { useEffect, useState } from 'react';
 import Api from '../Api';
 
 function QuestionsList() {
+  const { id: questionnaireId } = useParams();
```

Comment:
Here we look to see if there's an id in the path, and store it in a variable called questionnaireId... i.e. if the url path is something like `/questionnaires/[id]/questions`...